sycl/include/CL/sycl/INTEL/esimd/detail/emu/esimdcpu_device_interface.hpp

```diff
@@ -18,102 +18,11 @@
 
 #pragma once
 
-#include <CL/sycl/detail/pi.hpp>
+#include <CL/sycl/detail/defines_elementary.hpp>
 
-// cstdint-type fields such as 'uint32_t' are to be used in funtion
-// pointer table file ('esimd_emu_functions_v1.h') included in 'struct
-// ESIMDDeviceInterface' definition.
-#include <cstdint>
+__SYCL_WARNING(
+    "CL/sycl/INTEL/esimd/detail/emu/esimdcpu_device_interface.hpp usage is "
+    "deprecated, include "
+    "sycl/ext/intel/esimd/detail/emu/esimdcpu_device_interface.hpp instead")
```

Comment:
here and in similar places:
```suggestion
    "sycl/ext/intel/experimental/esimd/emu/detail/esimdcpu_device_interface.hpp instead")
```
see also comment about correct esimd files location below

Reply:
Done